controllers/kserve_inferenceservice_controller_test.go

```diff
@@ -19,56 +19,66 @@ import (
 	"context"
 	kservev1alpha1 "github.com/kserve/kserve/pkg/apis/serving/v1alpha1"
 	kservev1beta1 "github.com/kserve/kserve/pkg/apis/serving/v1beta1"
-	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
Should not this be a separate PR or you need some functionality?

Reply:
It could be, but as you see it's a rather minimal change due to backward comp of API. Also, note that this version was already pulled in as indirect dep.

It first came when I wanted to use `Ordered` decorator with `BeforeAll`, but gave up on that idea

Reply:
Though it comes in a separate commit (80af622ae9c8a09f199dfd602875d) so I can slice it if that's preferred. WDYT @Jooho @vaibhavjainwiz?

Reply:
Separate PR #147 

Reverted here in ad03002